go-server-server/go/default.go

```diff
@@ -280,8 +280,18 @@ func ConfigInterfaceVlanPost(w http.ResponseWriter, r *http.Request) {
         return
     }
     if vlan_kv != nil {
+        vlan_if_kv, _ := GetKVs(db.db_num, generateDBTableKey(db.separator, VLAN_INTF_TB, vlan_name))
+        if vlan_if_kv != nil {
+            if vnet_name, ok := vlan_if_kv["vnet_name"]; ok {
+                vnet_kv, _ := GetKVs(db.db_num, generateDBTableKey(db.separator, VNET_TB, vnet_name))
+                WriteRequestErrorWithSubCode(w, http.StatusConflict, RESRC_EXISTS,
+                    "Object already exists: {\"vlan_name\":\"" + vlan_name + "\", \"vnet_id\":\"" + 
+                    vnet_kv["guid"] +"\"}", []string{}, "")
+                return
+            }
```

Comment:
this else also we should return error message but with empty vnet_id

Reply:
which else are you referring to? if `vlan_kv` is nil then the rest of code block needs to be executed. We cannot return an error message. if `vlan_if_kv` is nil, we return the previous error message. If `vnet_name` key is missing in `vlan_if_kv`, then also we return the previous error message.

Reply:
i'm talking about if vnet_name == ""